xtask/codegen/css.ungram

```diff
@@ -463,6 +594,11 @@ CssGenericProperty =
 //  background: transparent center/1em auto no-repeat;
 //              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 // }
+//
+// This node type is implicitly added to any Node definition with the name
```

Comment:
Do you mean that this node type is added to any **value** Node definition?

What do you think about having a fallback for the **entire** property node to `CssGenericProperty`?

Reply:
I think having the fallbacks available on each specific property is nice because it ensures you can always query for a property purely by the struct in the AST. Like every declaration of `width`, whether it's valid, invalid, a default, or even has a bogus, syntactically incorrect value, will always be a `CssWidthProperty` node, and you can be confident that you won't miss any when traversing the tree. So it'd be like:

```css
width: 10px; /* CssWidthProperty, with CssRegularDimension as the value */
width: 10s; /* CssWidthProperty, with CssGenericComponentValueList as a value */
width: 10(\; /* CssWidthProperty, with BogusPropertyValue as a value */
unknown: 10s; /* CssGenericProperty, with CssGenericComponentValueList as a value */
--custom: 10s; /* CssGenericProperty, with CssGenericComponentValueList as a value */
``` 
I know we had `CssCustomProperty` for a little while that handle the `--custom` ones, and maybe that's worth bringing back to indicate that it's semantically valid and understood, separate from `CssGenericProperty` that would generally mean "this looks valid, but we don't know what the property is, so we can't enforce any semantics".

Unfortunately, either way if we do just fall back to generic for invalid values, we probably still want to do the automatic code generation stuff with the union to add `CssWideKeyword` as a value, since that applies to all properties, too. I think that's why I'm really more okay with having this be another variant on every property type, since there will already be at least 2 or 3 variants no matter what.